tests/e2e/tests/pilot/cloudfoundry/copilot_test.go

```diff
@@ -66,40 +64,8 @@ func pilotURL(path string) string {
 	}).String()
 }
 
-var routeRule = fmt.Sprintf(`
-apiVersion: config.istio.io/v1alpha2
-kind: VirtualService
-metadata:
-  name: route-for-myapp
-spec:
-  hosts:
-  - %s
-  gateways:
-  - cloudfoundry-ingress
-  http:
-  - route:
-    - destination:
-        name: example-app-guid.apps.cloudfoundry.internal
-`, publicRouteName)
-
-var routeRule2 = fmt.Sprintf(`
-apiVersion: config.istio.io/v1alpha2
-kind: VirtualService
-metadata:
-  name: route-for-other-app
-spec:
-  hosts:
-  - %s
-  gateways:
-  - cloudfoundry-ingress
-  http:
-  - route:
-    - destination:
-        name: example2-app-guid.apps.cloudfoundry.internal
-`, publicRouteName2)
-
 var gatewayConfig = fmt.Sprintf(`
-apiVersion: config.istio.io/v1alpha2
+apiVersion: config.istio.io/v1alpha3
```

Comment:
this is wrong I think.. check with @GregHanson / @frankbu .. Its networking.istio.io IIRC - recent change

Reply:
Yup, it's `networking.istio.io/v1alpha3`, thanks for catching that.